logging/bin/deploy_eventrouter.sh

```diff
@@ -23,17 +36,24 @@ set -e
 # Enable workload node placement?
 LOG_NODE_PLACEMENT_ENABLE=${LOG_NODE_PLACEMENT_ENABLE:-${NODE_PLACEMENT_ENABLE:-false}}
 
+# If performing an upgrade-in-place, check for/remove Event Router in kube-system namespace (if older than 1.1.3) 
+if [ ! -z "$V4M_CURRENT_VERSION_FULL" ]; then
```

Comment:
The comment above says this is only done if the version is older than 1.1.3, but the check only tests if V4M_CURRENT_VERSION_FULL is not empty.  Was that variable only added in 1.1.3?